src/main/java/codeu/model/data/User.java

```diff
@@ -106,4 +112,34 @@ public String getTime() {
     return date;
   }
 
+  /** When a User is initialized, this method creates a private Conversation with every other User that already exists. */
+  public void createConversations() {
```

Comment:
Does this need to be public? Seems like it could be package-private (to make it visible to the tests)

Reply:
Yeah it's probably better as private since it's only called by the User() constructor every time a new User is added -- fixed it!